shared/src/business/useCases/practitioners/updatePractitionerUserInteractor.test.js

```diff
@@ -1,46 +1,65 @@
 const {
   applicationContext,
 } = require('../../test/createTestApplicationContext');
+const {
+  COUNTRY_TYPES,
+  SERVICE_INDICATOR_TYPES,
+} = require('../../entities/EntityConstants');
 const {
   updatePractitionerUserInteractor,
 } = require('./updatePractitionerUserInteractor');
 const { ROLES } = require('../../entities/EntityConstants');
-const { SERVICE_INDICATOR_TYPES } = require('../../entities/EntityConstants');
 const { UnauthorizedError } = require('../../../errors/errors');
 jest.mock('../users/generateChangeOfAddress');
+const { generateChangeOfAddress } = require('../users/generateChangeOfAddress');
 
 describe('updatePractitionerUserInteractor', () => {
   let testUser;
-  let mockUser = {
-    admissionsDate: '2019-03-01T21:40:46.415Z',
-    admissionsStatus: 'Active',
-    barNumber: 'AB1111',
-    birthYear: 2019,
-    email: 'ab@example.com',
-    employer: 'Private',
-    firmName: 'GW Law Offices',
-    firstName: 'Test',
-    lastName: 'Attorney',
-    name: 'Test Attorney',
-    originalBarState: 'Oklahoma',
-    practitionerType: 'Attorney',
-    role: ROLES.privatePractitioner,
-    userId: 'df56e4f8-b302-46ec-b9b3-a6a5e2142092',
-  };
+  let mockPractitioner;
 
   beforeEach(() => {
     testUser = {
       role: ROLES.admissionsClerk,
       userId: 'admissionsclerk',
     };
 
+    mockPractitioner = {
```

Comment:
Why don't we make use of `shared/src/test/mockUsers.js` here?

Reply:
This mock practitioner has now been moved to `mockUsers`.